### ISSUE TITLE
Add has_prominent_author to github stats

### ISSUE THREAD

**Issue by orta** [CocoaPods/metrics.cocoapods.org#28]
When doing a look through the raw stats for the repo, see if the owner has more than 500 stars.

**Comment by segiddins** [comment]
Not great since it won't count organization stars.

**Comment by orta** [comment]
Nope, not great. Is it worth then trying to get the orgs average stars? Might be a bit too much effort for the metric.
